### DISCUSSION TITLE
Where to buy original ELM327 ?

### DISCUSSION THREAD

**Discussion by KarlBaumann** [cedricp/ddt4all#1267] [General]
It is really difficult to find the original ELM327 these days, since the clones are everywhere.

**Comment by shrlnm** [comment]
https://www.elmelectronics.com/

**Reply by KarlBaumann** [reply]
<img width="827" alt="image" src="https://github.com/user-attachments/assets/c4893470-17ee-4b0b-81f6-550afad2fc09">
